charts/shoot-core/components/charts/network-policies/templates/allow-to-public-networks.yaml

```diff
@@ -0,0 +1,21 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  annotations:
+    gardener.cloud/description: |
+      Allows Egress from pods labeled with 'networking.gardener.cloud/to-public-networks=allowed'
+      to all networks.
+  name: gardener.cloud--allow-to-public-networks
```

Comment:
```suggestion
  name: gardener.cloud:system:allow-to-public-networks
```

Reply:
`:` is an invalid character in networkpolicies

Reply:
Nice catch.
Then how about to remove the `gardener.cloud--` prefix ? Basically, you have already done it for the initial version of the PR. @rfranzke any objection?

Reply:
I think we should prefix the network policies with something so that users know they came from Gardener. `allow-to-dns` or `allow-to-public-networks` is very generic and it is likely that users already have or will have network policies with these names.

Reply:
+1, that was the purpose suggesting it.

Reply:
The label `origin: gardener` is enough to show that the network policy is from gardener. For name collisions - you are right, some users might have already deployed such resources. 

How about `allow-to-public-networks.gardener.cloud` or `gardener.cloud.allow-to-public-networks` ?

Reply:
I don't care. Just have `gardener.cloud` in the name to prevent conflicts.